api/views.py

```diff
@@ -345,6 +346,48 @@ def organization(request):
     return JsonResponse(data, safe=False)
 
 
+def companies(request):
+    """
+    Returns a query of customizable amount of companies (default= 10) based on user input
+    """
+    if request.method == "GET":
+        limit = request.GET.get("limit", 0)
+        input = request.GET.get("input", "")
+
+        limit = int(limit)
+
+        if limit == 0:
+            companies = Company.objects.filter(name__icontains=input)
+        else:
+            companies = Company.objects.filter(name__icontains=input)[:limit]
+
+        data = [serializers.companies(request, company) for company in companies]
+        return JsonResponse(data, safe=False)
+
+    else:
+        return HttpResponseBadRequest("Unsupported method!", content_type="text/plain")
+
+
+def status(request):
```

Comment:
I remember deleting this function/endpoint, it must have gotten readded in this PR. Could you delete it?

Reply:
No, I was referring to the `status` function on line 371.